generator/src/main/java/com/scottlogic/deg/generator/outputs/dataset_writers/CsvDataSetWriter.java

```diff
@@ -11,9 +11,12 @@
 import java.math.BigDecimal;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.stream.Collectors;
 
 public class CsvDataSetWriter implements DataSetWriter<CSVPrinter> {
+    private static final DateTimeFormatter standardDateFormat = DateTimeFormatter.ofPattern("dd-MM-yyyy hh:mm:ss");
```

Comment:
I think we should be Dependency injecting this everywhere, rather than repeating it